specification/hybridkubernetes/resource-manager/Microsoft.Kubernetes/preview/2020-01-01-preview/examples/ConnectedClustersListClusterCredentialsResult.json

```diff
@@ -3,7 +3,16 @@
     "subscriptionId": "1bfbb5d0-917e-4346-9026-1d3b344417f5",
     "resourceGroupName": "k8sc-rg",
     "clusterName": "testCluster",
-    "api-version": "2020-01-01-preview"
+    "api-version": "2020-01-01-preview",
+    "ClientAuthenticationDetails": {
+      "authenticationMethod": "Certificate",
```

Comment:
>Certificate [](start = 31, length = 11)

Can you add another example here and leave the default example with no body? No bodyworks for AAD scenario. For non AAD case user needs to pass in credential

Reply:
added